packages/components/accordion/src/AccordionItemTrigger.tsx

```diff
@@ -43,9 +43,12 @@ export const ItemTrigger = forwardRef<HTMLButtonElement, AccordionItemTriggerPro
         <div className="flex items-center gap-lg">{children}</div>
         <Icon
           intent="neutral"
-          className={cx('shrink-0 rotate-0 transition duration-100 ease-in', {
-            'rotate-180': isOpen,
-          })}
+          className={cx(
+            'shrink-0 rotate-0 transition duration-100 ease-in motion-reduce:!transition-none',
```

Comment:
Why use the `!` modifier here?

You can write like this instead:
```tsx
shrink-0 rotate-0 motion-reduce:transition duration-100 ease-in
```

Reply:
Wasn't aware of this `motion-safe`, I will do that and try.